hack/templates/00-hypershift-dataplane-metrics-forwarder.SelectorSyncSet.yaml.tmpl

```
@@ -172,7 +172,15 @@ objects:
                                   - effect: NoSchedule
                                     key: node-role.kubernetes.io/infra
                                     operator: Exists
-                                retention: 1d
+                                retention: 6d
```

As per the latest comment on the Jira, let's ensure parity with classic.

```suggestion
                                retention: 11d
```